ir/instr.cpp

```diff
@@ -1739,7 +1739,7 @@ StateValue Calloc::toSMT(State &s) const {
 
   expr is_null = p == Pointer::mkNullPointer(s.getMemory())();
   expr calloc_sz = expr::mkIf(is_null, expr::mkUInt(0, sz.bits()), size);
-  
+
```

Comment:
I guess Visual Studio Code is automatically remove the space whenever it is touched. :/
This is a good change anyway; I think convention is that empty line should not have space.